pkg/virt-launcher/virtwrap/converter/arch/converter_test.go

```diff
@@ -14,7 +14,7 @@
  *
  */
 
-package converter
+package arch
```

Comment:
Consider changing to `arch_test`.

Reply:
Thank you! This will result in exporting the different implementations of arch.`Converter`.
I think it is not worth it to expose them just for black box testing

Reply:
Fair point.